README.md

```diff
@@ -122,7 +122,13 @@ Also, we also provide Optuna docker images on [DockerHub](https://hub.docker.com
 
 ## Contribution
 
-Any contributions to Optuna are welcome! When you send a pull request, please follow the
+Any contributions to Optuna are welcome!
+
+If you are a new contributor, please check [good first issues](https://github.com/optuna/optuna/labels/good%20first%20issue). These are simple and well-defined so that they can be good starting points for you to get familiar with the contribution workflow and other developers.
+
+If you have already contributed to Optuna, I would highly recommend the other [contribution-welcome issues](https://github.com/optuna/optuna/labels/contribution-welcome).
+
+When you send a pull request, please follow the
 [contribution guide](./CONTRIBUTING.md).
```

Comment:
Although not introduced by this PR, this sentence sounds quite stiff with "please". How about something like the following. "General" is used to cover both of the above two cases (first time, not first time contributions).
```suggestion
For general guidelines how to contribute to the project, take a look at [CONTRIBUTING.md](./CONTRIBUTING.md).
```